bodo/pandas/vendor/duckdb/src/include/duckdb/storage/compression/chimp/algorithm/output_bit_stream.hpp

```diff
@@ -8,7 +8,6 @@
 
 #pragma once
 
-#include "duckdb.h"
```

Comment:
Maybe it's better to avoid removing duckdb.h and just modify it to minimize changes since it's imported in many places looks like. But not a big deal since the imports are not too many looks like.